Python/notify_me/notify-me.py

```diff
@@ -0,0 +1,45 @@
+from plyer import notification

```

Comment:
Add a GIF showing a pop-up notification

Reply:
@kaustubhgupta 
Might I clarify my understanding, the current commit folder has a .ico image file "Paperclip.ico" which is the one shows on the notification as below:
![run](https://lh3.googleusercontent.com/RjiZ5yjZu9UjpKrJPEyQSYMGV5X0f7KVSElXoYF3E_z__tsKcHb0Exh1vTJ6SH5bJfpW7OydGJgMhR_vEJCUazdUvcNFtsk7xK5geiJBwN7RtFuXMtkqmb7AZ0oXxzoPpVF2heSMgsOf4QIlqBOicYEC9TIWz9WqaoBtVAVePxdLe3vl5eRRKCLf0dbRxfpUDQ09nbdTy5Sg7njFMAbv8h4dfJlL800IHkAtgzsnnz9wgCIdoqIhwfHIRR7_YXBQXM2Vv2XqytQHuAfsSwJe8G-UIN4MlPAdjcGgQongO4uermYefY-R8cj4BvK4k9V99tscLQTwlL14M8CJEt1JO_CBb0mwxwA3seUC330wKgW0paEWD8LnQM0QjpPx5wwMd6uEzrRBQhh5K3iTHWBzDkgwa7eYIzKDgcWU0w5plB9EEbIMp_TGLi6-QWFCJyXkk6hvsDTYD5diQZzvIEvqNkP6aDSWYkIe7z-hntlDcxePg_2a1Iu8_-mhK5mOgGx3BXFqW1a7sMl78w6c4pKyDDdBf_axThNSStKQ9qOHC-qDTht0udqxnJTkVXP7rYfnuWSiN00G6aU7-Vf2szwfTCvdnrvP5Ylv7lh_Jhf22bC_hosSexE1Ids29-9H1hya6n3wtLfGn2FbiN8U41WLgKHFBWkMxmh6X0ytGP93LdYjO8T9gbe3nlV4Wl7V2qo=w752-h476-no?authuser=0)
![run](https://lh3.googleusercontent.com/gxcXoKiEO61f0IhgDpidUQwXCURZCAnbOt9Tq6DUWmofBobAKRbNFU9yKXnpyKZwz1b43Ow3NuPiuckv_18eK6FGTE83_iexNte6oHrvHSww-cLtwTIh11URIhSKNRZmdsnae_JE9H1KQkjZyVEEG2bvyNC2of5j8_KwQIefgnr6jg0Ttm6lD3nRaP7wcGj4R39G8EG1CPppYiHSslQLckj7OcT41YxuGLUJDRw-iy9yC9A-LTwItrN_7yQR-OZTdf55-zZHpjtZfkMbO3qIVI8Uwx6F9Hj2pEkXimPvv3Z9sZsYcJ5UJIMXXwXOklfGy0DgwiX-LCOfjyCQwNtmBBQNRdsHgViA_3aZNGSYAKMvlIokt8WhLY2aJuhSR_VFH-ciPsi9tsBtAY32wtpyiFQ1Gb7ZWc2no9Lmp79nAaiQT3GDpyalmZQDnMhYUeS7b2Nk7TtHZZTnOkeuBq0RNvADWfbF6IpWZdEK-c6JIaV0mGtEMXzoGWIiYKgUYvU5SivMJKchvwVtybMS_GGrneuSort7qorW_Yrkp8--ww7yGTpnvmaTdI2Wtlw39yD4ErUrR3utP4lVSFz91XGBHTpr8s-yh5qh9RXIGk0yZHxGkyp-QzYewxMV1GS1Bvx3piye8BmUXAxn0J1nTZLcaT27meRD7kMOGFv7vXov6vZc-XVGI7H9wHl7jvporJg=w746-h228-no?authuser=0)
Do you mean that the .ico should be changed to .gif file?

Reply:
I think the file name is confusing. Paperclip.ico is an output screenshot or logo for notifications?

Reply:
@kaustubhgupta  thanks for the comment.
.ico file is "icon" image file extension for microsoft windows. The [definition](https://en.wikipedia.org/wiki/ICO_(file_format)) for your reference.
Please kindly let me know if:
- [ ] it is allowed to keep  current format, 
- [ ] or needs to be modified to .gif file.

Reply:
@kaustubhgupta I double checked on the source code of plyer. 
the notification icon for windows seems only allow to use .ico extension.
Is it ok to keep current format? or should I change the file name of the .ico file?
The source code screenshot below for your reference: 
![image](https://user-images.githubusercontent.com/52155607/96211619-cb557180-0faf-11eb-880d-319d123bbe8d.png)

Reply:
@kaustubhgupta I think I might misunderstand your word. I had changed the file name of .ico file to notification_icon.
Please kindly help to double confirm if it is ok